libs/island-ui/core/src/lib/Select/Select.css.ts

```diff
@@ -258,13 +258,13 @@ export const optionDescriptionTruncated = style({
 export const optionSizes = styleVariants({
   xs: wrapMedia(inputMixins.inputSizes.xs, `${wrapper} &`),
   sm: wrapMedia(inputMixins.inputSizes.sm, `${wrapper} &`),
-  md: wrapMedia(inputMixins.inputSizes.md, `${wrapper} &`),
```

Comment:
Could you please explain this change? Seems kind of weird that `sm` and `md` are the same.

Reply:
right, the design' been changed for select dropdown options to be 18px instead of 24px as it looks pretty big. I've changed it here to test it out. Not sure if i should keep this change or make a different approach?

Reply:
Ah, I get it now. I think this approach is fine then..